### PR TITLE
clean up profile header and info table

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
This works in some feedback that @ncksllvn had left on a previous PR https://github.com/department-of-veterans-affairs/vets-website/pull/12461

## Testing done
Local; confirmed everything still looks good!

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 1]
```diff
@@ -85,23 +85,30 @@ const ProfileHeader = ({
...
-        <div
-          className={[
-            ...serviceBadgeClasses,
-            ...serviceBadgeClassesMedium,
-            'usa-width-one-fourth',
-          ].join(' ')}
-        >
+    <div className={classes.wrapper}>
+      <div className={classes.innerWrapper}>
+        <div className={classes.serviceBadge}>
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:111]: I'm _pretty_ sure that we can't get away from this mild case of divitis. They are mostly there to handle layout concerns.

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 2]
```diff
@@ -85,23 +85,30 @@ const ProfileHeader = ({
...
+      ...serviceBadgeClasses,
+      ...serviceBadgeClassesMedium,
+      'usa-width-one-fourth',
+    ].join(' '),
+    title: [...titleClasses, ...titleClassesMedium].join(' '),
+    fullName: [...fullNameClasses, ...fullNameClassesMedium].join(' '),
+    latestBranch: [...latestBranchClasses, ...latestBranchClassesMedium].join(
+      ' ',
+    ),
+  };
```
Comment (ncksllvn) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:106]: I love this! I'd still consider moving it to the parent scope outside of the component as constants, or do you prefer it here? From my very limited experience with it, `react-native` usually does that with its style definitions

**Review by ncksllvn — APPROVED** [review]
Looks great!

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx (hunk 2 above)
Comment (erikphansen) [reply src/applications/personalization/profile-2/components/ProfileHeader.jsx:106]: I don't have super strong feelings about this one way or another. I guess the benefits of defining them outside of the component would be that it'd be easier to test those constants if you wanted to and also be able to more easily swap out different style objects. Anyway this is a huge improvement, IMO. Thanks for the suggestion.

**Pull request merged**